### PR TITLE
Spelling enterprise

### PR DESCRIPTION
**Pull request by jsoref**
This is split from #35947

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: tests/snuba/api/endpoints/test_organization_sessions.py [hunk 1]
```diff
@@ -135,16 +135,16 @@ def test_inaccessible_project(self):
...
-        assert response.data == {"detail": 'Invalid field: "summ(sessin)"'}
+        assert response.data == {"detail": 'Invalid field: "summ(session)"'}
 
     def test_unknown_groupby(self):
-        response = self.do_request({"field": ["sum(session)"], "groupBy": ["envriomnent"]})
+        response = self.do_request({"field": ["sum(session)"], "groupBy": ["environment"]})
 
         assert response.status_code == 400, response.content
-        assert response.data == {"detail": 'Invalid groupBy: "envriomnent"'}
+        assert response.data == {"detail": 'Invalid groupBy: "environment"'}
```
Comment (priscilawebdev) [inline tests/snuba/api/endpoints/test_organization_sessions.py:147]:
```suggestion
        assert response.data == {"detail": 'Invalid groupBy: "_environment"'}
```

File: tests/snuba/api/endpoints/test_organization_sessions.py [hunk 2]
```diff
@@ -135,16 +135,16 @@ def test_inaccessible_project(self):
...
-        response = self.do_request({"field": ["summ(sessin)"]})
+        response = self.do_request({"field": ["summ(session)"]})
 
         assert response.status_code == 400, response.content
-        assert response.data == {"detail": 'Invalid field: "summ(sessin)"'}
+        assert response.data == {"detail": 'Invalid field: "summ(session)"'}
 
     def test_unknown_groupby(self):
-        response = self.do_request({"field": ["sum(session)"], "groupBy": ["envriomnent"]})
+        response = self.do_request({"field": ["sum(session)"], "groupBy": ["environment"]})
```
Comment (priscilawebdev) [inline tests/snuba/api/endpoints/test_organization_sessions.py:144]:
```suggestion
        response = self.do_request({"field": ["sum(session)"], "groupBy": ["_environment"]})
```

Comment (priscilawebdev) [inline tests/snuba/api/endpoints/test_organization_sessions.py:144]: I believe this has to be invalid

**Comment by priscilawebdev** [comment]
@jsoref you have conflicts 😉

File: tests/snuba/api/endpoints/test_organization_sessions.py (hunk 1 above)
Comment (jsoref) [reply tests/snuba/api/endpoints/test_organization_sessions.py]: Looks like this was resolved in 8b13ba0e0 (sorry, these splits aren't quite as systematic as I've done for other projects which were strict splittable directories....)

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**